python/tk_desktop/desktop_window.py

```diff
@@ -284,6 +282,59 @@ def _pop_dll_state(self):
                 engine = sgtk.platform.current_engine()
                 engine.log_warning('Could not restore DllDirectory under Windows.')
 
+    def register_tab(self, tab_name, tab_widget):
+        '''
```

Comment:
@robblau I used the file convention in this case. Should I change the whole file at the same time?